### ISSUE TITLE
Doesnt work for Blender 2.9.1

### ISSUE THREAD

**Issue by furbzeyyygaming** [AlwarrenSidh/ArmAToolbox#36]
![image](https://user-images.githubusercontent.com/70483614/126067382-21f0427d-e764-4a5f-b7da-5c6c8a90dae0.png)

**Comment by AlwarrenSidh** [comment]
This looks like an installation problem, make sure you reinstall the correct version

**Comment by LegalSmeagol** [comment]
I don't think this is an installation problem. On a fresh install of Blender, I get this issue persistently. I've tried another reinstall of Blender and Arma Toolbox just to be certain and I still get this issue.
